src/adaptermanager.js

```diff
@@ -211,6 +214,65 @@ exports.makeBidRequests = function(adUnits, auctionStart, auctionId, cbTimeout,
   return bidRequests;
 };
 
+function isValidBidRequest(adUnits) {
+  adUnits.forEach(adUnit => {
+    if (adUnit.sizes) {
+      utils.logWarn('Usage of adUnits.sizes will eventually be deprecated.  Please define size dimensions within the corresponding area of the mediaTypes.<object> (eg mediaTypes.banner.sizes).');
+    }
+
+    if (adUnit.mediaTypes) {
+      if (adUnit.mediaTypes.banner) {
+        if (adUnit.mediaTypes.banner.sizes) {
+          adUnit.sizes = adUnit.mediaTypes.banner.sizes;
+        } else {
+          utils.logError('Detected an adUnits.mediaTypes.banner object did not include sizes.  Removing adUnits.mediaTypes.banner object from bid request');
+          delete adUnit.mediaTypes.banner;
+        }
+      }
+
+      if (adUnit.mediaTypes.video) {
+        if (adUnit.mediaTypes.video.playerSize) {
+          if (
+            Array.isArray(adUnit.mediaTypes.video.playerSize) &&
+            adUnit.mediaTypes.video.playerSize.length === 2 &&
+            !isNaN(adUnit.mediaTypes.video.playerSize[0])) {
+            adUnit.sizes = adUnit.mediaTypes.video.playerSize;
+          } else {
+            utils.logError('Detected incorrect configuration of mediaTypes.video.playerSize.  Please specify only one set of dimensions in a format like: [640, 480]');
```

Comment:
when you delete something (video in this case) make sure you add that into the warn message. I.E. "`mediaType.video` has been removed from this request. "

Reply:
I'll update the messages accordingly.